hw/ip/clkmgr/dv/cov/clkmgr_manual_cov_unr_excl.el

```diff
@@ -0,0 +1,49 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+//
+// This contains some obvious exclusions the UNR tool didn't flag.
+
+//==================================================
+// This file contains the Excluded objects
+// Generated By User: maturana
+// Format Version: 2
+// Date: Thu Sep 29 13:57:41 2022
+// ExclMode: default
+//==================================================
+CHECKSUM: "2301929872 1660332954"
+INSTANCE: tb.dut.u_clk_main_aes_trans.u_idle_cnt
+ANNOTATION: "[UNSUPPORTED] Input tied to a constant, and unr doesn't detect it."
```

Comment:
```suggestion
ANNOTATION: "[UNR] Input tied to a constant."
```

Aren't these UNR?

Reply:
I see... I had it as UNR in an ancient PR that tried this, but then I saw some exclusion file stating UNSUPPORTED for a case like this and I assumed the writer was better informed...I assume an annotation change doesn't invalidate the checksum, since it is not in the RTL. Will update it.

Reply:
Done.